Pod/Classes/MenuView.swift

```diff
@@ -8,6 +8,8 @@
 
 import UIKit
 
+internal let MenuViewItemsMarginRigth: CGFloat = 20
```

Comment:
Hi Kitasuke. First of all thanks for your framework.

So I declared it that way so it could be easy to change later if the right margin was not the desired. Because of some constants like MinimumSupportedViewCount, in that moment I thought it will make sense. Honestly looking at it right now probably will be better to just have a constant inside the same function contentOffsetXForCurrentPage. 
Let me know what you think. 

Thanks !